tests/php/test-class-amp-base-sanitizer.php

```diff
@@ -270,6 +271,46 @@ public function test_remove_invalid_child() {
 		AMP_Validation_Manager::$validation_results = null;
 	}
 
+	/**
+	 * Tests remove_invalid_child.
+	 *
+	 * @covers AMP_Base_Sanitizer::remove_invalid_child()
+	 * @covers AMP_Base_Sanitizer::has_dev_mode_exemption()
+	 */
+	public function test_remove_invalid_child_dev_mode() {
+		$html = '<div></div>';
```

Comment:
Instead of a faceless div, a fetch by class and an indexed access, followed by a docblock to type-hint to an element, I'd suggest adding an ID to the div and retrieving a direct `DOMElement` by ID:
```php
$html = '<div id="testee"></div>';
// [...]
$element = $dom->getElementById( 'testee' ); // <-- immediately type-hinted as DOMElement
```
This is shorter, faster, and doesn't require the extra docblock for type-hinting.